src/pymodule/dense/plot/plot_structures.py

```diff
@@ -397,7 +472,7 @@ def plot_dendrogram(neurite, axis=None, show_node_id=False,
     tips = set(tree.tips)
 
     # if diameter is ignored, set all values to default_diam
-    default_diam = vertical_diam_frac*vspace
+    default_diam = 0.2*vspace
```

Comment:
I'm pretty sure ``vertical_diam_frac`` is necessary, was there a good reason to change this?